app/src/Components/AddRole/AddRolePopUp.tsx

```diff
@@ -0,0 +1,14 @@
+interface AddRolePopUpProps {
+    message: string;
+    color: string;
+}
+
+const AddRolePopUp = ({ message, color }: AddRolePopUpProps) => {
```

Comment:
Please use a variable instead of an anonym object. You can use the 'props' variable name and access it's properties like with an object (props.message and pops.color)